Unified/rejector.py

```diff
@@ -240,39 +254,45 @@ def rejector(url, specific, options=None):
                     schema['Multicore'] = mcore
                     schema['Memory'] = mem
                 print json.dumps( schema, indent=2 )
-                newWorkflow = reqMgrClient.submitWorkflow(url, schema)
-                if not newWorkflow:
-                    msg = "Error in cloning {}".format(wfo.name)
-                    print(msg)
-                    wfi.sendLog('rejector',msg)
-                          
-                    # Get the error message
-                    time.sleep(5)
-                    data = reqMgrClient.requestManagerPost(url, "/reqmgr2/data/request", schema)
-                    wfi.sendLog('rejector',data)
-                    
-                    print json.dumps( schema, indent=2 )
-                    return 
-                print newWorkflow
+                if not options.test:
+                    newWorkflow = reqMgrClient.submitWorkflow(url, schema)
+                    if not newWorkflow:
+                        msg = "Error in cloning {}".format(wfo.name)
+                        print(msg)
+                        wfi.sendLog('rejector',msg)
 
-                data = reqMgrClient.setWorkflowApproved(url, newWorkflow)
-                print data
-                wfi.sendLog('rejector','Cloned into %s by unified operator %s'%( newWorkflow, comment ))
-                #wfi.notifyRequestor('Cloned into %s by unified operator %s'%( newWorkflow, comment ),do_batch=False)
+                        # Get the error message
+                        time.sleep(5)
+                        data = reqMgrClient.requestManagerPost(url, "/reqmgr2/data/request", schema)
+                        wfi.sendLog('rejector',data)
+
+                        print json.dumps( schema, indent=2 )
+                        return 
+                        print newWorkflow
+
+                    data = reqMgrClient.setWorkflowApproved(url, newWorkflow)
+                    print data
+                    wfi.sendLog('rejector','Cloned into %s by unified operator %s'%( newWorkflow, comment ))
+                    #wfi.notifyRequestor('Cloned into %s by unified operator %s'%( newWorkflow, comment ),do_batch=False)
             else:
-                wfo.status = 'trouble' if options.set_trouble else 'forget' 
-                wfi.notifyRequestor('Rejected by unified operator %s'%( comment ),do_batch=False)
-                session.commit()
+                if options.test:
+                    print 'Rejected by unified operator %s'%( comment )
+                else:
+                    wfo.status = 'trouble' if options.set_trouble else 'forget' 
+                    wfi.notifyRequestor('Rejected by unified operator %s'%( comment ),do_batch=False)
+                    session.commit()
 
         else:
```

Comment:
and this one too might need formatting of spaces.

Reply:
I think it's in the right place, i.e. it's "else" for "if all(results)".